go/cpufreqgetter/README.md

```diff
@@ -0,0 +1,6 @@
+# cpufreqgetter
+C and golang code to build minimal images to get read and dump data from /proc/cpuinfo
```

Comment:
This is just go code

Reply:
I updated the readme. We don't need the c version. All of @irfanurrehman 's suggestions have been implemented in the Go code.